src/gui/advancedsettings.h

```diff
@@ -62,7 +63,10 @@ private slots:
     QSpinBox spin_cache, spin_save_resume_data_interval, outgoing_ports_min, outgoing_ports_max, spin_list_refresh, spin_maxhalfopen, spin_tracker_port, spin_cache_ttl;
     QCheckBox cb_os_cache, cb_recheck_completed, cb_resolve_countries, cb_resolve_hosts, cb_super_seeding,
               cb_program_notifications, cb_torrent_added_notifications, cb_tracker_favicon, cb_tracker_status,
-              cb_confirm_torrent_recheck, cb_enable_tracker_ext, cb_listen_ipv6, cb_announce_all_trackers;
+              cb_confirm_torrent_recheck, cb_listen_ipv6, cb_announce_all_trackers;
+#if LIBTORRENT_VERSION_NUM < 10200
```

Comment:
Could anyone suggest what would be a better (according to the project standards/rules) way to do this? Referencing a libtorrent include file from the GUI part is wrong, but I can't a see a solution better than putting this into the qBt `config.h` file and referencing it instead of `LIBTORRENT_VERSION_NUM`.